tools/crmadmin.c

```diff
@@ -526,8 +528,6 @@ main(int argc, char **argv)
     g_strfreev(processed_args);
     g_clear_error(&error);
     pcmk__free_arg_context(context);
-    free(options.status);
-    free(options.kill);
```

Comment:
We still need to free these, but with g_free()

In practice it doesn't matter since the command is exiting at this point, but it helps make valgrind less noisy if we use it on this

Reply:
I cleaned them out of the equation (options consists of 2 booleans and an int now) :)

Reply:
Whoops I'm getting sloppy :) Trying to rush too much ...